Fordpass Widget.js

```diff
@@ -52,8 +52,9 @@ const widgetConfig = {
     debugMode: false, // ENABLES MORE LOGGING... ONLY Use it if you have problems with the widget!
     refreshInterval: 5, // allow data to refresh every (xx) minutes
     useIndicators: true, // indicators for fuel bar
-    unitOfLength: userData.useMetric ? "km" : "mi", // unit of length
+    unitOfLength: userData.useMetric ? 'km' : 'mi', // unit of length
     distanceMultiplier: userData.useMetric ? 1 : 0.621371, // distance multiplier
+    unitOfPressure: 'psi', // unit of pressure, default from ford is kpa
```

Comment:
Thanks for catching this but a new variable isn't required... I just fixed the pressureToFixed() function to remove .toFixed